x-pack/plugins/infra/server/plugin.ts

```diff
@@ -97,6 +97,14 @@ export const config: PluginConfigDescriptor<InfraConfig> = {
         ),
       })
     ),
+    featureFlags: offeringBasedSchema({
+      traditional: schema.object({
+        metricsExplorerEnabled: schema.boolean({ defaultValue: true }),
```

Comment:
Tied to the comment I left in `core_plugins/rendering.ts`...
I'm wondering if we could move the use of `offeringBasedSchema` down to the individual flag level. This would still allow you to define a different serverless default for `metricsExplorerEnabled`, but would also allow each (current and future) exposed setting under `featureFlags` to be included in testing.